src/pcloud/api.py

```diff
@@ -38,9 +38,18 @@ def main():
 
 class PyCloud(object):
 
-    endpoint = "https://api.pcloud.com/"
+    O_WRITE = int("0x0002", 16)
+    O_CREAT = int("0x0040", 16)
+    O_EXCL = int("0x0080", 16)
+    O_TRUNC = int("0x0200", 16)
+    O_APPEND = int("0x0400", 16)
+
+    def __init__(self, username, password, eu_server=False):
+        if eu_server:
+            self.endpoint = "https://eapi.pcloud.com/"
+        else:
+            self.endpoint = "https://api.pcloud.com/"
```

Comment:
The idea is not bad, but it makes the the tests fail and it does not scale. How about a endpoint dict with (api, eapi, unittest) as keys and the according URLs. This way it is easy to add more and the tests won't break.

Reply:
This way `pycloud` will work with all endpoints and you don't have to make changes at all:
```
def __init__(self, username, password, endpoint="api"):
    self.endpoint = "https://" + endpoint + ".pcloud.com/"
```